datafusion/core/tests/sql/joins.rs

```diff
@@ -1413,6 +1413,8 @@ async fn hash_join_with_dictionary() -> Result<()> {
 }
 
 #[tokio::test]
+#[ignore]
+// https://github.com/apache/arrow-datafusion/issues/3565
```

Comment:
skip the test,  https://github.com/apache/arrow-datafusion/issues/3565
@AssHero want to take it

Reply:
the issue has been fixed, so I remove the comment and `#[ignore]`

Reply:
https://github.com/apache/arrow-datafusion/pull/3636/commits/824ad9975f45ec32475fb59a2ef7e190049f0908